README.md

```diff
@@ -368,10 +367,10 @@ url: `https://dashboard.syncano.io/#/instances/${tempInstance.instanceName}/scri
 [...]
 ```
 
-First line is same as in test, we simply `import` our file with data used for tests. As for second line, you will see that we are referring to `instanceName` from `tempInstance.js` file.
-Thanks to that our tests is universal, every time new test instance will be created, we don't need to change the url. Our navigate function used in tests will know where to go.
+First line is the same as in `scriptEndpointsPage.js` file. We simply `import` our file with data used for the tests. As for second line, you will see that we are referring to `instanceName` from `tempInstance.js` file.
+Thanks to that our tests are universal. Every time a new test instance is be created, we don't need to change the url. Our navigate function used in tests will know where to go.
 
-> You may also see that I have used different locators that are used in Part I and Part II, this is due to that we are rewriting some parts of dashboard to include data-e2e attribute to easily and without any issue target DOM objects on website while testing them. They are written in css selector, omitting type of tag that they are attached to. DOM can change easliy but with selectors like this we don't need to maintain test that much. But this whole concept is more for future parts.
+> You may also see that I have used different locators that are used in Part I and Part II. This is due to the fact, that we are rewriting some parts of the Dashboard to include data-e2e attributes. This helps us targeting DOM objects easily and without any issues while performing tests. Locators are written as css selectors, omitting type of tag that they are attached to. DOM can change easliy but with selectors like this we're spending less time on test maintenance. We'll discuss this approach in the future blog posts.
 
 So now we are ready to run tests using our tool and newly created tests.
 To do that, you could use every command one by one, but this way you can forget some of the steps.
```

Comment:
` babel-node scripts/createTestData.js \
   && npm test tests/testScriptEndpoint.js \
   && babel-node scripts/cleanUp.js` this could land in package.json as `test` command.